web/styles/zulip.css

```diff
@@ -3295,3 +3295,86 @@ select.invite-as {
         padding: 5px;
     }
 }
+
+#add-poll-modal {
+    /* on wide screens, this height allows roughly 3 option
+    rows to fit in without need for scrolling */
+    height: 450px;
+    overflow: hidden;
+
+    @media (width < $sm_min) {
+        height: 80vh;
+    }
+
+    .modal__content {
+        flex-grow: 1;
+
+        .simplebar-content {
+            box-sizing: border-box;
+            height: 100%;
+            display: flex;
+            flex-direction: column;
+        }
+    }
+
+    #add-poll-form {
+        display: flex;
+        flex-direction: column;
```

Comment:
Strictly speaking, this columnar flex seems necessary. But you can still get by without the one I mentioned above, I think.

Reply:
Yes, this one needs to be retained since without it, the scrollable list of options won't work